salt/modules/mount.py

```diff
@@ -432,6 +434,141 @@ def match(self, line):
         return True
 
 
+class _filesystems_entry(object):
```

Comment:
Class names are typically camel-case: https://www.python.org/dev/peps/pep-0008/#class-names

Reply:
Yeah, and they don't need to be private. It's a common misconception that it'll be picked up by Sphinx autodoc but it only picks up functions, not classes.

Reply:
@terminalmage @cachedout I was following the current code's implementation, _fstab_entry, _vfstab_entry. I can certainly change the name, however it would appear inconsistent with existing code.  I prefer to maintain the style and usage already present in the file, but if preferred I can update my additions to be conformant with existing standards.  
Let me know which is the preference moving forward so that I can be consistent in the future.

Reply:
Updated to CamelCase